src/components/accessories/table/Table.tsx

```diff
@@ -147,6 +164,17 @@ const Table: FunctionComponent<IProps> = ({
       ) : (
         ""
       )}
+
+      <ConfirmationDialog
+        isOpen={openDeleteConfirmation}
+        title={t("common.delete")}
+        info={t("common.deleteconfirmation")}
+        icon={warningIcon}
+        primaryButtonLabel="OK"
+        secondaryButtonLabel="Dismiss"
+        handlePrimaryButtonClick={handleDelete}
+        handleSecondaryButtonClick={() => setOpenDeleteConfirmation(false)}
+      />
```

Comment:
I reviewed your work and I think that the component's logic is OK.

The only thing i've noticed is that: maybe in order to be more exaustive, we should add the content's title with the confirmation message eg: Are you sure to delete the #title_of_the_row ?

Reply:
Ok.